coretex/_task/_local.py

```diff
@@ -120,11 +123,12 @@ def configure(self) -> None:
 
     @classmethod
     def _readTaskRunConfig(cls) -> List[BaseParameter]:
-        if not os.path.exists("experiment.config"):
-            raise FileNotFoundError(">> [Coretex] \"experiment.config\" file not found")
-
         parameters: List[BaseParameter] = []
-        with open("./experiment.config", "rb") as configFile:
+
+        if not os.path.exists(EXPERIMENT_CONGIF_PATH):
+            return []
+
+        with open(EXPERIMENT_CONGIF_PATH, "rb") as configFile:
```

Comment:
After updating `EXPERIMENT_CONGIF_PATH` to be a `pathlib.Path` we can directly call `exists` and `open` methods on it.